### DISCUSSION TITLE
How to maintain the order of the parameters on the return

### DISCUSSION THREAD

**Discussion by gsmatheus** [prisma/prisma#14888] [Q&A]
I need the output to be ordered in the same order as the parameters, in a query MySql it would look like this

```mysql
SELECT * FROM your_table
WHERE id IN (5,2,6,8,12,1)
ORDER BY FIELD(id,5,2,6,8,12,1);
```

How can I do it using the prism?

**Comment by austincrim** [comment]
Hey @gsmatheus 👋,

This is not possible yet with a plain Prisma query. We do have an [outstanding feature request](https://github.com/prisma/prisma/issues/9708). Feel free to add a 👍 or your specific use case.

In the meantime, you can use [`$queryRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
